website/src/components/brand-switcher/brand-switcher.js

```diff
@@ -71,8 +71,8 @@ export const BrandSwitcher = () => {
 
 	return (
 		<Fragment>
-			<div css={{ height: 35, padding: SPACING(2) }}>
-				<Link href={`/?b=${brandName}`}>
+			<div css={{ height: 35, padding: `${SPACING(6)} ${SPACING(4)}` }}>
+				<Link href={`/design-system?b=${brandName}`}>
```

Comment:
These links won't work. Next needs to link to a page template. Should be `<Link href={BASE_PAGE} as={`${BASE_URL}?b=${brandName}`}>`